core/src/block.rs

```diff
@@ -19,894 +19,630 @@ use iroha_data_model::{block::*, events::prelude::*, transaction::prelude::*};
 use iroha_version::{declare_versioned_with_scale, version_with_scale};
 use parity_scale_codec::{Decode, Encode};
 
-pub use self::{
-    candidate::{CandidateBlock, VersionedCandidateBlock},
-    candidate_committed::{CandidateCommittedBlock, VersionedCandidateCommittedBlock},
-    chained::ChainedBlock,
-    pending::{PendingBlock, VersionedPendingBlock},
-    signed::SignedBlock,
-    valid::ValidBlock,
-};
 use crate::{
     prelude::*,
     sumeragi::network_topology::{Role, Topology},
     tx::TransactionValidator,
 };
 
-mod pending {
-    use super::*;
+declare_versioned_with_scale!(VersionedPendingBlock 1..2, Debug, Clone, iroha_macro::FromVariant);
+
+/// Transaction data is permanently recorded in chunks called
+/// blocks.
+#[version_with_scale(n = 1, versioned = "VersionedPendingBlock")]
+#[derive(Debug, Clone, Decode, Encode)]
+pub struct PendingBlock {
```

Comment:
What about just `Block`?

Reply:
`PendingBlock` is much more informative.